pkgs/by-name/et/etl/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, meson
+}:
+
+stdenv.mkDerivation rec {
+  pname = "etl";
+  version = "20.38.6";
```

```suggestion
  version = "20.39.4";
```
New version released by now